runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -10041,11 +9932,6 @@ bool J9::X86::TreeEvaluator::VMinlineCallEvaluator(
             }
             break;
 
-         case TR::java_lang_Math_sqrt:
-         case TR::java_lang_StrictMath_sqrt:
```

Comment:
There is also a similar piece in Power codegen which we need to remove [1].

[1] https://github.com/eclipse/openj9/blob/6887baea4c98f2820f42071c29de5f7a7ce31642/runtime/compiler/p/codegen/J9TreeEvaluator.cpp#L12863-L12870